src/modules/75-cf/components/FlagArchiving/useArchiveFlagDialog.tsx

```diff
@@ -46,36 +107,65 @@ const useArchiveFlagDialog = ({
     }
   }
 
+  const getDialogContent = () => {
+    if (state === STATUS.error)
+      return <PageError message={showError(dependentFlagsError)} onClick={() => refetchDependentFlags()} />
```

Comment:
`message` is expected to be what's shown to the user. `showError` is a function that shows the toaster. They're not really compatible. Either we should show the error using `PageError` or we should show the toaster, not both. In this case, since it's the fetch and display, I'd just show the error and ditch the toaster.
```suggestion
      return <PageError message={dependentFlagsError} onClick={() => refetchDependentFlags()} />
```

Reply:
this must have been causing the duplicate error messages appearing in the test suites